### DISCUSSION TITLE
json file for playground produces visual error

### DISCUSSION THREAD

**Discussion by hennihaus** [graphql-java-kickstart/graphql-spring-boot#615] [Q&A]
Hello together,

i have made a json file to show an example call in the graphql playground. Unfortunately the graphql playground shows an error, although all variables are equal. Is this any bug or have I made a mistake? The query delivers a correctly answer.

```
      - name: Journey query
        query: classpath:playground/requests/get-journeys-query-max-parameters.graphql
        variables: classpath:playground/variables/journey-variables.json
```

![image](https://user-images.githubusercontent.com/47771386/110841774-9de2ca80-82a6-11eb-9194-51da7ee1e890.png)

**Comment by BlasiusSecundus** [comment]
If the query runs correctly, it is most likely not your mistake. This is possibly an issue in Playground code itself. If you try manually enter/copy-paste the query, does this issue occur?
